docs/modules/ROOT/pages/runbooks/KyvernoAdmissionLatencyReachedWebhookTimeout.adoc

```diff
@@ -0,0 +1,14 @@
+= Alert rule: KyvernoAdmissionLatencyReachedWebhookTimeout
+
+include::partial$runbooks/contribution_note.adoc[]
+
+== icon:glasses[] Overview
+
+Kyverno average admission latency is higher than the webhook timeout.
+Creating or updating Kubernetes object failed because of the admission latency.
+
+== icon:bug[] Steps for debugging
+
+* Check Kyverno memory and CPU usage / limits.
+* Check Kyverno logs
+* VSHN: Notify Tarazed
```

Comment:
I'm missing steps for addressing the cause of the alert: does it help to increase memory and CPU limits? What are common error messages and what can I do to fix them?

Reply:
We don't really know. I guess I should add the client side throttling log message. It's the only one we know.

Reply:
I added a note about the client side throttling.